docs/extensions/reference/_posts/1970-01-06-CLI.md

```diff
@@ -423,6 +324,25 @@ By default, `shoutem pack` command will do `npm run build` in both `app` and `se
 
 Run this command from extension folder.
 
+- ##### shoutem configure [--release]
+
+Runs platform's configure script to sync the local app with native changes to local extensions.
+The flag --release is used to switch the local app to the last published configuration on the server.
+Call the `shoutem configure` command again without the flag to switch back to the current development configuration.
+
+The command should be called when:
+- switching between published (shoutem configure --release) and development (shoutem configure) configuration
+- changing native code of any of the extensions from the extensions/ directory
+- changing cocoapods or gradle dependencies of any of the extensions from the extensions/ directory
+- new extension is created, installed to the current app & pushed
```

Comment:
This is the first and most common reason! Put this order:

- new extension is created, installed to the current app & pushed
- changing native code of any of the extensions from the extensions/ directory
- changing cocoapods or gradle dependencies of any of the extensions from the extensions/ directory
- switching between published (`shoutem configure --release`) and development (`shoutem configure`) configuration

Also, please use backticks.